legacy/ui/legacy/src/main/java/com/fsck/k9/ui/messagelist/MessageListAdapter.kt

```diff
@@ -533,12 +533,11 @@ class MessageListAdapter internal constructor(
     }
 
     private fun selectForegroundColor(selected: Boolean, read: Boolean, active: Boolean): Int {
-        val backGroundAsReadIndicator = appearance.backGroundAsReadIndicator
```

Comment:
This makes the setting `Settings` -> `General Settings` -> `Change colour when read` ineffective. Therefore, it should be removed to avoid confusing users with a non-functional option.